File: R3/R03.js
const FindOutSecoundsIn = (hours, minute) =>{

    const OneHourHave = 60; //Minutes
    const OneMinuteHave = 60; //Seconds

    let TotalMinutes = (hours * OneHourHave) + minute;
    let TotalSeconds = TotalMinutes * OneMinuteHave

    console.log(TotalSeconds)//116400
}

FindOutSecoundsIn(32,20);

File: R3/R03.py
def FindOutSecoundsIn(hour,minutes):
    one_hour_have = 60;
    one_minute_have = 60;

    total_minutes = (hour * one_hour_have)+minutes;
    total_seconds = (total_minutes * one_minute_have);

    print(total_seconds)


FindOutSecoundsIn(32,20)
